### ISSUE TITLE
[QUESTION] one output case

### ISSUE THREAD

**Issue by DeidaraYang** [Harry24k/adversarial-attacks-pytorch#134]
### ❔ Any questions

my model only has one output followed by a sigmoid function (0.5 as threshold to determine the label is 1 or 0). but it seems like these api only support multiple output followed by a softmax function. How can I modify it to support our model senario?

**Comment by rikonaka** [comment]
Hi @DeidaraYang , it doesn't matter how many dimensions of output you have, it doesn't affect the work of torchattacks, but you have to make sure the ground truth label and the output are in the same dimension.
